configuration/configuration.go

```diff
@@ -265,6 +285,10 @@ func (c *Data) GetAuthURL() string {
 	return c.v.GetString(varAuthURL)
 }
 
+func (c *Data) SetAuthURL(value string) {
```

Comment:
not needed, if you need to set in tests, then you can do an `os.SetEnv(..)` and then re-initialize the config object

Reply:
removed, done!